posthog/api/routing.py

```diff
@@ -43,8 +46,19 @@ class StructuredViewSetMixin(_GenericViewSet):
         JwtAuthentication,
         PersonalAPIKeyAuthentication,
         authentication.SessionAuthentication,
-        authentication.BasicAuthentication,
     ]
+    additional_authentication_classes = []
+
+    permission_classes = [IsAuthenticated, OrganizationMemberPermissions, TeamMemberAccessPermission]
```

Comment:
`TeamMemberAccessPermission` only applies to endpoints scoped by team

Reply:
Ah, you mentioned "only validating the team access if the endpoint is itself a team-scoped endpoint" – but then why have it at all?

Reply:
So what I meant is "why have it on non-`is_team_view`" viewsets. But both work the same, so either way works